src/app/fyle/spender-onboarding/spender-onboarding.page.ts

```
@@ -1,5 +1,5 @@
 import { Component } from '@angular/core';
-import { finalize, forkJoin, from, map, Observable, switchMap } from 'rxjs';
+import { catchError, finalize, forkJoin, from, map, Observable, switchMap } from 'rxjs';
```

ESLint

[error] 'catchError' is defined but never used.

(@typescript-eslint/no-unused-vars)